gm4_shapeless_portals/data/gm4_shapeless_portals/functions/x/check_location.mcfunction

```diff
@@ -0,0 +1,7 @@
+# check if new location is valid for a portal frame
+# @s = new portal frame marker
+# at @s
```

Comment:
`at @s` isn't very useful here. Perhaps change it to something like "at a potential new portal block"